openedx/core/djangoapps/auth_exchange/tests/utils.py

```diff
@@ -2,10 +2,16 @@
 Test utilities for OAuth access token exchange
 """
 
-from social.apps.django_app.default.models import UserSocialAuth
+from django.conf import settings
+from social_django.models import UserSocialAuth, Partial
+
 from third_party_auth.tests.utils import ThirdPartyOAuthTestMixin
 
 
+TPA_FEATURES_KEY = 'ENABLE_THIRD_PARTY_AUTH'
+TPA_FEATURE_ENABLED = TPA_FEATURES_KEY in settings.FEATURES
```

Comment:
@bradenmacdonald That's intentional, but I may be under the wrong impression: is the platform trying to slowly sway code in `common/djangoapps` to `openedx/core`? Because I wanted to stop referring directly to the old version as much as I could for new code.